static/js/src/advantage/offers/Offers.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Row, Col } from "@canonical/react-components";
-import OffersList from "./components/OffersList";
+import OffersList from "./components/OffersList/OffersList";
```

Comment:
You could just the index file to load this to keep it tidy

Reply:
VSCode tried to be smart here